$MirOS: ports/mailnews/pine/patches/patch-pine_pine_hlp,v 1.1.7.1 2005/03/18 15:49:48 tg Exp $
$OpenBSD: patch-pine_pine_hlp,v 1.4 2003/10/05 21:42:29 jakob Exp $
--- pine/pine.hlp.orig	Thu Apr 28 17:21:40 2005
+++ pine/pine.hlp	Wed May  4 23:28:11 2005
@@ -306,10 +306,11 @@ Additions included:
 <P>
 
 <UL>
+  <LI> The file &quot;.pinerc&quot; is located in the &quot;.etc/&quot; subdirectory of the user's home directory now.
   <LI> The &quot;^T&quot; &quot;To Files&quot; command from within the composer on the Attchmnt header line or from within the composer in the &quot;^J&quot; &quot;Attach&quot; command is now capable of attaching multiple files at once
   <LI> When Exporting a message with attachments you now have the option of saving all of the attachments in a new directory (<A HREF="h_ge_allparts">Export Message File Selection</A>) by typing &quot;^P&quot; while exporting
   <LI> Feature to modify behavior of TAB command, <A HREF="h_config_tab_uses_unseen">Tab-Uses-Unseen-For-Next-Folder</A>
-  <LI> When sending, flowed text (<A HREF="http://www.ietf.org/rfc/rfc3676.txt">RFC 3676</A>) is sent by default. Now it is possible to turn off this behavior on a message by message basis with the ^V command at the Send prompt. You may still turn if off for all cases with the feature <A HREF="h_config_quell_flowed_text">Quell-Flowed-Text</A>.
+  <LI> When sending, flowed text (<A HREF="http://www.ietf.org/rfc/rfc3676.txt">RFC 3676</A>) is sent by default. Now it is possible to turn off this behavior on a message by message basis with the ^V command at the Send prompt. You may still turn if off for all cases with the feature <A HREF="h_config_quell_flowed_text">Quell-Flowed-Text</A>; this is recommended, and suggested through the code.
   <LI> <A HREF="h_config_viewer_margin_left">Viewer-Margin-Left</A> and <A HREF="h_config_viewer_margin_right">Viewer-Margin-Right</A> made a little more flexible
   <LI> Suppress quoted text when viewing messages with <A HREF="h_config_quote_suppression">Quote-Suppression-Threshold</A>
   <LI> The behavior of the <A HREF="h_common_hdrmode">HdrMode Command</A> has changed slightly. It now reverts back to normal mode when you switch to a new message. It also has an additional state if <A HREF="h_config_quote_suppression">Quote-Suppression-Threshold</A> is turned on.
@@ -959,14 +960,14 @@ PC-Pine:
 Unix Pine:
 
  executable     &lt;Unix search path&gt;/pine
- persnl cfg     ~/.pinerc
- except cfg     ~/.pinercex
+ persnl cfg     ~/.etc/pinerc
+ except cfg     ~/.etc/.pinercex
  global cfg     <!--#echo var="PINE_CONF_PATH"-->
  fixed cfg      <!--#echo var="PINE_CONF_FIXED_PATH"-->
  local help     <!--#echo var="PINE_INFO_PATH"-->
 
  interrupted    ~/.pine-interrupted-mail
- debug          ~/.pine-debugN
+ debug          ~/.etc/pine-debugN
  crash		~/.pine-crash
  newsrc#        ~/.newsrc
  signature#     &lt;pinerc dir&gt;/.signature
@@ -975,8 +976,7 @@ Unix Pine:
  postponed#     ~/mail/postponed-msgs
  sentmail#      ~/mail/sent-mail
  mailcap#       ~/.mailcap + /etc/mailcap
-                           + /usr/etc/mailcap + /usr/local/etc/mailcap
- mimetypes#     ~/.mime.types + /etc/mime.types + /usr/local/lib/mime.types
+ mimetypes#     ~/.mime.types + /etc/mime.types + /etc/mime.types
 
  news-spool	varies across Unix flavors, e.g. /var/spool/news or /usr/spool/news
  active-news	varies across Unix flavors, e.g. /usr/lib/news/active
@@ -16774,7 +16774,8 @@ configuration file) throughout Pine, or 
 You can't mix the two.
 <P>
 This field consists of a filename which will be used as the signature
-file when using this role.
+file when using this role. If the filename is not an absolute pathname,
+it will be rooted in the same directory as your pinerc.
 <P>
 If the filename is followed by a vertical bar (|) then instead
 of reading the contents of the file the file is assumed to be a
@@ -19258,7 +19259,7 @@ a semi-colon (;) under Windows; for exam
 </PRE>
 <!--chtml else-->
 a colon (:) under UNIX; for example:<PRE>
-        ~/.mailcap:/etc/mailcap:/usr/etc/mailcap:/usr/local/etc/mailcap
+        ~/.mailcap:/etc/mailcap
 </PRE>
 <!--chtml endif-->
 <P><UL>
@@ -20067,7 +20068,7 @@ PINERC file is a local file.
 If your PINERC file is remote, then it will be in the directory specified
 by the &quot;-aux local_directory&quot; command line option.
 <!--chtml else-->
-&quot;.signature&quot;.
+&quot;.signature&quot; in the same directory as your pinerc.
 <!--chtml endif-->
 <P>
 
@@ -20083,6 +20084,8 @@ program which will produce the text to b
 The program can't have any arguments and doesn't receive any input from Pine,
 but the rest of the processing works as if the contents came from a file.
 
+If the filename is not an absolute pathname, it will be
+rooted in the same directory as your pinerc.
 <P>
 Instead of storing the data in a local file, the
 signature data may be stored remotely in an IMAP folder.
@@ -24496,7 +24499,7 @@ argument (see the &quot;lynx&quot; examp
 So, here are some example entries:
 <PRE>
 URL-Viewers = _TEST(&quot;test -n '$&#123;DISPLAY}'&quot;)_  /usr/local/bin/netscape
-              &quot;/usr/local/bin/lynx _URL_&quot;
+              &quot;/usr/bin/lynx _URL_&quot;
               C:&#92;BIN&#92;NETSCAPE.BAT
 </PRE>
 <P>
@@ -24504,7 +24507,7 @@ This example shows that for the first vi
 the environment variable &quot;DISPLAY&quot; must be defined.  If it
 is, then the path and file &quot;/usr/local/bin/netscape&quot; must exist.  
 If neither condition is met,
-then the path and file &quot;/usr/local/bin/lynx&quot; must exist.  
+then the path and file &quot;/usr/bin/lynx&quot; must exist.  
 If it does, then the &quot;_URL_&quot; token is replaced by the selected URL. 
 If the path to &quot;lynx&quot; is invalid,
 then the final path and file C:&#92;BIN&#92;NETSCAPE.BAT must exist.  
@@ -24600,7 +24603,7 @@ because of its length, but should all ap
 URL-Viewers = _TEST("test -L /myhomedir/.netscape/lock")_ &quot;/usr/local/bin/netscape -remote 'openURL(_URL_,  new-window)' &amp;&quot;<BR>
 
 _TEST(&quot;test -n '$&#123;DISPLAY}'&quot;)_ &quot;/usr/local/bin/netscape &amp;&quot;<BR>
-              &quot;/usr/local/bin/lynx '_URL_'&quot;
+              &quot;/usr/bin/lynx '_URL_'&quot;
 <P>
 <!--chtml endif-->
 <P>
